include/TimeseriesStats.h

```diff
@@ -30,6 +30,7 @@ class TimeseriesStats: public GenericTrafficElement {
   Host *host;
   u_int32_t total_alerts;
   u_int32_t unreachable_flows_as_client, unreachable_flows_as_server;
+  u_int32_t port_unreachable_flows_as_client, port_unreachable_flows_as_server;
```

Comment:
The point is that actually unreachable_flows_as_client and unreachable_flows_as_server were already containing the port-unreachable-flows of this pull request. 

I would leave  u_int32_t unreachable_flows_as_client, unreachable_flows_as_server.